en/hardware/reference_design.md

```diff
@@ -27,10 +27,14 @@ The diagram below shows the division of bus and functional responsibilities betw
 
 <!-- Draw.io version of file can be found here: https://drive.google.com/file/d/1H0nK7Ufo979BE9EBjJ_ccVx3fcsilPS3/view?usp=sharing -->
 
-The I/0 board is optional!
 Some Pixhawk-series controllers are built without the I/O board in order to reduce space or complexity, or to better address certain board use-cases.
-An I/O board is required for [airframes](../airframes/airframe_reference.md) that use AUX ports for motors or control surfaces; generally it is not needed for multicopters or for *fully* autonomous vehicles (without a safety pilot using RC control).
+An I/O board is *required* for [airframes](../airframes/airframe_reference.md) that use AUX ports for motors or control surfaces; generally it is not needed for multicopters or for *fully* autonomous vehicles (without a safety pilot using RC control).
 
-> **Note** Manufacturers often create flight controller variants with and without the I/O board. 
-  The version without an I/O board is usually a size-reduced version that is named as a "diminutive" of the version that has both boards.
+The I/0 board is disabled by unsetting the [SYS_USE_IO](../advanced/parameter_reference.md#SYS_USE_IO) parameter.
```

Comment:
Would "setting `SYS_USE_IO` to `0`" be more clear than "unsetting"?

Reply:
Very minor nit-pick: here you have "I/0" (with a zero), elsewhere you have "I/O" (the letter).

Reply:
Yes to both.